openedx/features/edly_panel/api/v1/views.py

```diff
@@ -0,0 +1,13 @@
+from rest_framework import status
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+
+
+@api_view(['GET'])
+def get_maus(request):
```

Comment:
We should avoid using abbreviations and technical jargon in code, we can rename this view to `get_monthly_active_users`

Reply:
done.